src/features/home/pages/ThematicHome.native.test.tsx

```diff
@@ -59,7 +52,7 @@ describe('ThematicHome', () => {
     renderThematicHome()
     await act(async () => {})
 
-    expect(await screen.findAllByText('Bloc temps fort')).toBeTruthy()
+    expect(screen.getAllByText('Bloc temps fort')).toBeTruthy()
```

Comment:
screen.getAllByText retourne une liste, donc sera toujours truthy : il vaut mieux tester not.toHaveLength(0) ou qqch du genre ?

Reply:
j'ai retesté en mettant des caractères random, et il me mettait bien faux, qu'il ne trouvait pas les éléments avec le texte random

Reply:
getAllByText va throw une erreur s'il ne trouve aucun élément